packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -453,7 +453,7 @@ const RealAccountSignup = ({
                 active_modal_index_no = modal_pages_indices.add_or_manage_account;
             } else {
                 active_modal_index_no =
-                    !currency && real_account_signup_target !== 'maltainvest'
+                    (!currency && real_account_signup_target !== 'maltainvest') || is_loading
```

Comment:
too many rerenders causes this function to be called on each render due to this the index was being reset and another form was being rendered. 
checking for loading state prevents resetting if the form is in submitting state